### ISSUE TITLE
Feature/Help can we set for permanent link

### ISSUE THREAD

**Issue by dipenpatel235** [ignacionelson/ProjectSend#414]
for example my file name is hello.pdf
and i want to download as https://localhost/xyz/hello.pdf    like that ?

if i will upgrade hello.pdf file then i get latest file from same above link.

Let me know if that type of existing feature available or not.
i have try to find that bit not seen. thats why i have file ticket.

Thanks
Dipen

**Comment by kkplein** [comment]
Perhaps this would be possible with the new (in progress) "public groups" functionality. If there were a 'landing page' for a group, that url would be your url, and you would make the group contain the file hello.pdf

**Comment by dipenpatel235** [comment]
means this feature in on development right ?

**Comment by kkplein** [comment]
yes, I am just unsure if the per-group-landing-page will be implemented also...

**Comment by dipenpatel235** [comment]
Ok..thanks... waiting​ for that feature.
Thanks.

**Comment by lenamtl** [comment]
Could you elaborate why you need this feature...

**Comment by ignacionelson** [comment]
Hello!
There will be a landing page PER GROUP. But I still haven't decided if it will be best to have a directory like page where you can navigate through all the public groups or just have it limited to a single one.
What do you think will be more usable?

**Comment by heupink** [comment]
I'm not sure I completely understand what you mean with the two options....

**Comment by kkplein** [comment]
You mean:
#1 have a global 'listing page' ("directory page") with an overview of all available public groups on a server..
or
#2 make each public group have it's own specific landing page?
That second option would be nice. With room for some info (text/contact info/logo/...) on that specific group..?
Or do I misunderstand..?

**Comment by ignacionelson** [comment]
^ Exactly like that.

#1 would be a 2 step navigation. one general landing page that lists all public groups, and you can click on each one to view its files. (I picture an UI like Google Drive's navigation).

#2 is just the second part of #1, a landing for a group with its files only. Adding more information would be great, I'll work on that.

**Comment by kkplein** [comment]
For us, option #1 would not be very relevant, but #2 is! :-) But that's just us...

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
